### ISSUE TITLE
Automate EPP ingestion

### ISSUE THREAD

**Issue by QueenKraken** [elifesciences/enhanced-preprints-server#452]
Description:

As an eLife Production team member I want reviews to automatically be published to save manual effort

# Acceptance Criteria

- [ ] Get the MECA archive from BioRxiv bucket
- [ ] Extract & ingest the XML
- [ ] Extract & ingest the images
- [ ] Extract & ingest the supplementary files

**Comment by QueenKraken** [comment]
@BlueReZZ is the plan still have Sciety message us when an eLife article is produced?

**Comment by BlueReZZ** [comment]
No, but that could be an option. An available option is to poll the DocMaps index, filtered by eLife as the group, and the last known published review date, and new ones will appear automatically.
